packages/generator/src/entity/class.spec.ts

```diff
@@ -43,4 +53,23 @@ describe('entity class generator', () => {
       [`${toBrunch.instancePropertyName}?`, 'Brunch<T> | null']
     ]);
   });
+
+  it('has expected number of methods', () => {
+    expect(classDeclaration.methods?.length).toEqual(2);
+  });
+
+  // TODO decide if list of parameters func(p1,p2,...) or like for unbound and currently implemented func({paramsObj},
+  xit('has expected parameters for bound function', () => {
+    const fn = classDeclaration.methods?.find(
+      ({ name }) => name === 'getPrice'
+    );
+
+    expect(fn?.parameters?.map(({ name }) => name)).toEqual(['meal']);
+  });
+
+  // TODO decide if list of parameters func(p1,p2,...) or like for unbound and currently implemented func({paramsObj},
+  xit('has expected parameters for bound action', () => {
```

Comment:
[req] left over?

Reply:
Yes. @FrankEssenberger I've tried to update the test but I'm not sure how I can actually get the parameters here.

Reply:
I think we aligned to keep the object for consistency in the call yesterday. In this case you have to check there is a paramter with name `paramaters` and the the type is `GetPriceParameterType` or however it is called. Then to make it 100% you would check the definition of the `GetPriceParameterType` that the object contains two properties. For this you have to check the `parametersInterfac()` result. Perhaps it is enough to check here the type and the two properties in the internal `statements` of the class and check if the `parametersInterface()` has a sparate test.